Bottom: fba6c9c590cddc149edad0d6054c96229eaefc13
Top:    51e443ade0a4f88b19ed694f2e8b1c5d9eaf1440
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-05-31 10:22:18 +0200

fix array access


---

diff --git a/objectformatter/src/main/scala/net/virtualvoid/string/ESParser.scala b/objectformatter/src/main/scala/net/virtualvoid/string/ESParser.scala
index 0a94924..d7f4d09 100644
--- a/objectformatter/src/main/scala/net/virtualvoid/string/ESParser.scala
+++ b/objectformatter/src/main/scala/net/virtualvoid/string/ESParser.scala
@@ -51,7 +51,8 @@ object AST{
     def format(o:AnyRef) = exp.eval(o) match{
       // array or collection or similar
     case l : java.lang.Iterable[AnyRef] => realEval(l)
-    case l : Seq[AnyRef] => realEval(l)
+    case l : Iterable[AnyRef] => realEval(l)
+    case a: Array[AnyRef] => realEval(a)
     }
   }
